articles/2016-09-15-units.md

```diff
@@ -0,0 +1,86 @@
+# :eyes: Units
+
+## Pixels
+
+A pixel is considered the smallest base unit, representing a single dot in an image. Every digital image is made up of a grid of pixels.
+
+Here is an example:
+
+![pixel example](../images/pixel.png)
+*Image courtesy of [wikipedia.](https://en.wikipedia.org/wiki/Pixel)*
+
+Generally when you're declaring the length of something in CSS, it will be using pixels (or `px`).
+
+## Percentage
+
+Percentages are *technically* not a unit of measurement, but they are the second most commonly used method to declare the length (or width, or margin, or anything) so it should still be discussed.
+
+When an element has `width: 50%;`, its width will be 50% of its direct parent's. If its a top-level element (meaning it has no container other than `<body>`), its width will be 50% of the window.
```

Comment:
I'd add a quick tidbit that browsers calculate percentages into pixels. If you grab the width of an image in the dom through js for example, you'll get a super precise px amount.